src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -1240,7 +1240,7 @@ public static bool IsGraphQLReservedName(Entity entity, string databaseColumnNam
             string schemaName,
             string tableName)
         {
-            DataTable? dataTable = EntitiesDataSet.Tables[tableName];
+            DataTable? dataTable = EntitiesDataSet.Tables[$"{schemaName}.{tableName}"];
```

Comment:
For MySQL, schemaName will be empty so this will turn into ".tablename", do we need a separate implementation of this function for MySQL similar to how